app/Http/Controllers/Admin/EDU/Program/ProgramController.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace App\Http\Controllers\Admin\EDU\Program;
```

Comment:
Is it program or programme?
I think british is programme in relation to e.g. a course/study. And program in terms of a computer program/software dev programming

Dunna matter either way I guess as long as we're consistent

Reply:
Lol, great point 😆 . I was too tired and tried to get the bulk of it done. Will go back and change it to programme.